### DISCUSSION TITLE
Help get Red Markers on items!

### DISCUSSION THREAD

**Discussion by jarlengrip** [Lailloken/Exile-UI#206] [General]
I cant get the red markers working when identifying items, as soon I right click again while holding shift it just abort the identifying going on and no red marker is marked. I have activited this in options but cant get a solution for this. I thingk i have setup everything correctly and the skript is running with admin rights.

Do you have any idea why it is not working? I really am impressed by this mod so far!

Thanks in advance
/Jarlengrip

**Comment by Lailloken** [comment]
There is an "inventory" pixel-check that can be calibrated in the "screen-checks" section of the settings menu. Have you done that? It enables the tool to read the screen and check the state of the inventory, then automatically close the tooltip and its markers if the inventory is closed.

It sounds like the tool is misreading the state of your inventory, which hints at inproper calibration. Try recalibrating the pixel-check.

**Reply by jarlengrip** [reply]
Thanks alot for the reply, after some trial and error with the pixel-checks i got it to be working. I guess i have wrongly saved the gamescreen because it worked after I recalibrated that one and then updated the inventory, not just the inventory.
